os/mm/Kconfig

```diff
@@ -146,3 +146,11 @@ config MM_SHM
 	---help---
 		Build in support for the shared memory interfaces shmget(), shmat(),
 		shmctl(), and shmdt().
+
+config BOOTMEM
+	bool "Direct memory allocation during boot up"
+	default y
```

Comment:
keeping this flag enabled by default conflicts with having this feature implemented for only MM_KERNEL_USRHEAP_INIT flag.  If it has dependency, we can enable/disable the required flags